kolibri/core/assets/src/views/AppError/index.vue

```diff
@@ -51,9 +49,30 @@
 
 <script>
 
+  import get from 'lodash/get';
+  import urls from 'kolibri.urls';
   import { mapActions } from 'vuex';
   import ReportErrorModal from './ReportErrorModal';
 
+  // Inspects URL and returns an enum of the plugin (enum values are "learn", "coach", etc.)
+  function getCurrentKolibriPlugin() {
+    const url = window.location.href;
+
+    if (url.includes(urls['kolibri:kolibri.plugins.learn:learn']())) {
```

Comment:
These checks will break if any of these plugins is disabled. Should always do a check that the URL exists:
`urls['kolibri:kolibri.plugins.learn:learn'] && url.includes(urls['kolibri:kolibri.plugins.learn:learn']())`